bindings/java/CMakeLists.txt

```diff
@@ -141,9 +141,35 @@ target_include_directories(java_workloads PUBLIC ${JNI_INCLUDE_DIRS})
 
 set(CMAKE_JAVA_COMPILE_FLAGS "-source" "1.8" "-target" "1.8")
 set(CMAKE_JNI_TARGET TRUE)
-set(JAR_VERSION "${FDB_MAJOR}.${FDB_MINOR}.${FDB_REVISION}")
+
+# build a manifest file
+set(JAR_VERSION "${FDB_MAJOR}.${FDB_MINOR}.${FDB_PATCH}")
+string(TIMESTAMP BND_LAST_MODIFIED_SEC "%s")
+set(MANIFEST_TEXT "Manifest-Version: 1.0
+Bnd-LastModified: ${BND_LAST_MODIFIED_SEC}000
+Build-Jdk: ${Java_VERSION_STRING}
+Built-By: CMake ${CMAKE_VERSION}
+Bundle-Description: FoundationDB java binding and API
+Bundle-ManifestVersion: 2
+Bundle-Name: fdb-java
+Bundle-SymbolicName: com.foundationdb
+Bundle-Version: ${JAR_VERSION}
+Created-By: CMake ${CMAKE_VERSION}
+Name: com.foundationdb
```

Comment:
This became com.apple.foundationdb , right ?

Reply:
Yep, just updated that and the one a few lines above.  Thanks!

Reply:
Looks like it wasn't pushed

Reply:
Ah, nevermind; I somehow got an old version.

But should it be `com/apple/foundationdb/`? I couldn't actually tell from the manifest docs what the right format is.

Reply:
You're right, that is the right format. But also, upon closer inspection, I don't think that the name attribute is supposed to be in this section anyway, so I removed it.  https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html.